index.js

```diff
@@ -55,6 +55,10 @@ module.exports.expressGAuth = function expressGAuth(options) {
         next(err)
       } else {
         passportSession(req, res, err => {
+          const indexOfQuestionMarkInOriginalUrl = req.originalUrl.indexOf('?')
+          const reqOriginalUrlWithoutQueryParams = indexOfQuestionMarkInOriginalUrl === -1 ?
+            req.originalUrl :
+            req.originalUrl.substring(0, indexOfQuestionMarkInOriginalUrl)
```

Comment:
URLs are nasty things where all manner of sicko stuff is valid. Would it perhaps be all around simpler to use [`new URL()`](https://nodejs.org/api/url.html#constructing-a-url-from-component-parts-and-getting-the-constructed-string) to achieve this same effect, instead of looking for the question mark in the string? Seems like combining the origin and pathname properties should achieve the same effect.

Reply:
In this case, there's no protocol/host available in `req.originalUrl` -- what would be idiomatic use of `new URL` for this..?

Reply:
Ah. Bugger. And we can't reasonably assume it'll always be http/s? 🤔

Reply:
Ah, missed that the host is missing too. Okay. Then I don't think there's really that many smarter ways to do this. You could perhaps do `req.originalUrl.split('?')[0]` if you wanted it a bit more terse, but that seems about it.

Reply:
Ok, I will put:

`const [ indexOfQuestionMarkInOriginalUrl ] = req.originalUrl.split('?')`

Reply:
That doesn't seem like it would work, as .split() will return a string, not a number. 🤔 Correct usage seems like it would be
```js
const [reqOriginalUrlWithoutQueryParams] = req.originalUrl.split('?')
```

Reply:
You were too quick :) -- I didn't have time to fix my incorrect param name there... ;)